recipes/ms-gsl/all/conanfile.py

```diff
@@ -70,14 +70,13 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "Microsoft.GSL")
-        self.cpp_info.set_property("cmake_target_name", "Microsoft.GSL")
 
         self.cpp_info.filenames["cmake_find_package"] = "Microsoft.GSL"
         self.cpp_info.filenames["cmake_find_package_multi"] = "Microsoft.GSL"
         self.cpp_info.names["cmake_find_package"] = "Microsoft.GSL"
         self.cpp_info.names["cmake_find_package_multi"] = "Microsoft.GSL"
 
-        self.cpp_info.components["_ms-gsl"].set_property("cmake_target_name", "GSL")
+        self.cpp_info.components["_ms-gsl"].set_property("cmake_target_name", "Microsoft.GSL::GSL")
```

Comment:
For this case in which the `_ms-gsl` component is just a workaround to set the target name, I think it's better to take the property to the root cpp_info instead of maintaining the component. Like here: https://github.com/conan-io/conan-center-index/pull/8281/files
Maybe we can close this one and leave the other PR.

Reply:
That sounds good to me unless anyone else has any objections.

Reply:
Closing in favor of #8281